Unix/doc/setup-ntlm-omi.md

```diff
@@ -0,0 +1,163 @@
+#Enabling SPNEGO/NTLM Authentication on OMI
+
+OMI requests between the client and server have been up to this point
+authenticated via Basic auth. Basic is very simple, but not at all secure, 
+inasmuch as the username and password are transmitted in the clear, with only 
+Base64 encoding which is easly decoded by someone able to see the traffic on the 
+connection. For that reason we only recommend Basic never be used on other than a 
+secure (https) conneciton.
+
+A more secure method of authentication uses Secure Protected Negotitation protocol, which enables
+negotiation of the security protcol from a selection of options, Currently the only option available with
+omi is the* NT Lan manager protocol, version 2 (NTLMV2)* which is an improvment over Basic auth in two ways. 
+
+- The password is hashed using an irreversable algorithm, so the password is more secure than Basic.
+
+- The client and server support encryption of your data over http connections, so SSL certificates are not required. 
+
+In order to use SPNEGO with NTLM, you will need to set up both the client and the server 
+to provide the credentials. 
+
+Note, NTLM support on gss is not available on MacOS. Solaris, Hpux, Aix or older linux distros
```

Comment:
Improper abbreviations.  Aix is usually `AIX` (all upper-case), Hpux is usually `HP-UX`. I'm not as sure on this, but isn't Linux generally capitalized first letter?